core/src/main/scala/org/broadinstitute/dsde/rawls/webservice/BillingApiServiceV2.scala

```diff
@@ -85,7 +87,14 @@ trait BillingApiServiceV2 extends UserInfoDirectives {
               get {
                 complete {
                   import spray.json._
-                  userServiceConstructor(ctx).getBillingProject(RawlsBillingProjectName(projectId)).map {
+
+                  // Check if the projectId is a UUID or a billingProjectName
```

Comment:
I didn’t think about the overlap between the existing `getBillingProject` and new `getBillingProjectById` APIs. My concern with this approach is that I could create a new billing project with a UUID for a name that overlapped with an existing billing project's id which would result in my new billing project being inaccessible to me. 

Instead, how does `GET /api/billing/v2/id/{billingProjectUUID}` sound?

Reply:
Fixed: https://github.com/broadinstitute/rawls/pull/3205/commits/3205dfcbb24ab052c6166be8b3845bcdbc6728ee